client/components/mma/switch/SwitchContainer.tsx

```diff
@@ -30,6 +36,17 @@ export interface SwitchContextInterface {
 	productDetail: ProductDetail;
 	isFromApp: boolean;
 	user?: MembersDataApiUser;
+	mainPlan: PaidSubscriptionPlan;
+	monthlyOrAnnual: 'Monthly' | 'Annual';
+	supporterPlusTitle: string;
+	thresholds: Thresholds;
+}
+
+export interface Thresholds {
+	monthlyThreshold: number;
+	annualThreshold: number;
+	chosenThreshold: number;
+	aboveThreshold: boolean;
```

Comment:
Minor detail, but I wonder if it would be clearer if this property was renamed to distinguish it from the other 3 similarly named properties which are all threshold amounts? `isAboveThreshold` or similar?

Reply:
isAboveThreshold is better 👍 not sure 'chosenThreshold' makes that much sense since a user can't choose between monthly or annual in this flow

Reply:
It's essentially the threshold for the billing period, so maybe something like `billingPeriodThreshold`? (Not sure if that makes any more sense 🤷‍♂️ )

Reply:
Or maybe we don't even need it as it's only used in a single place? That neatly avoids having to come up with a good name 😅

Reply:
is it? I thought that threshold was used in a number of places 🤔

Reply:
Only seems to be one reference to it in this PR in `SwitchComplete`.